app/views/statuses/_group.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="status-group">
+  <%= render 'statuses/status', status: group %>
+
+  <div class="status-items">
```

Comment:
You are right. I think the BEM introduction is an ongoing effort by @vernalkick. Unfortunately he have very little time to put on shipit.

I'm really bad at CSS and design, so I can't really finish the redesign project, but I'll try to find some time to improve the classes naming consistency.